devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy

```diff
@@ -273,6 +273,7 @@ def initialPoll() {
 	}
 	request << zwave.batteryV1.batteryGet()
 	request << zwave.sensorBinaryV2.sensorBinaryGet(sensorType: 0x0C) //motion
+	request << zwave.notificationV3.notificationGet(notificationType: 0x07, event: 0x08) //motion for Everspiring
```

Comment:
Technically, we are using Notification CC V3 here and the event "0x07: Motion Detection" is for V2 of the command class and later and event "0x08: Motion Detection" was added in V4.

My preference would be to use to 0x07, but odes the everSpring motion sensor respond to that instead of 0x08?

Reply:
When I set 0x07 device still pairing without motion status. With 0x08 this issue doesn't occur.

Reply:
Strange implementation.